tests/test_reduce.py

```diff
@@ -39,22 +65,66 @@ def test_count_2d_dt():
     assert dfg.topython() == [[None, 0, 1, 2, 3, 5, 8, 9],
                               [0, 1, 1, 1, 2, 2, 2, 1],
                               [3, 1, 1, 1, 2, 2, 1, 1],
-                              [3, 1, 1, 1, 2, 2, 2, 1]]                        
+                              [3, 1, 1, 1, 2, 2, 2, 1]]
 
 
-def test_count_2d_array():
-    a = [[9, 8, 2, 3, None, None, 3, 0, 5, 5, 8, None, 1],
-         [0, 1, 0, 5, 3, 8, 1, 0, 2, 5, None, 8, 1]]
+def test_count_array_string():
+    a = [None, "blue", "green", "indico", None, None, "orange", "red", "violet", "yellow", "green", None, "blue"]
```

Comment:
Please try not to write lines > 80 chars (unless it actually makes the code more readable).

Reply:
Sure, forgot to adjust my editor for that. Done.